t/11_hacluster.t

```diff
@@ -631,8 +632,9 @@ subtest '[crm_resource_meta_set] Delete meta-argument' => sub {
     my $hacluster = Test::MockModule->new('hacluster', no_auto => 1);
     my @calls;
     $hacluster->redefine(assert_script_run => sub { @calls = @_; return; });
+    $hacluster->redefine(record_info => sub { note(join(' ', 'RECORD_INFO -->', @_)); });
 
-    crm_resource_meta_set(resource => 'Hogwarts', meta_argument => 'RoomOfRequirement');
+    crm_resource_meta_set(resource => 'Hogwarts', meta_argument => 'RoomOfRequirement', argument_value => undef);
```

Comment:
Set explicitly to undef for clarity? Otherwise, it can (and  probably should irl usage) just omit the argument